ddcommon-ffi/src/vec.rs

```diff
@@ -21,6 +21,10 @@ pub struct Vec<T: Sized> {
     _marker: PhantomData<T>,
 }
 
+unsafe impl<T: Send> Send for Vec<T> {}
+
+unsafe impl<T: Sync> Sync for Vec<T> {}
```

Comment:
These were required for `datadog_profiling_ffi::Error` to be able to be converted to certain other Error types.

Reply:
This looks correct to me